config.go

```diff
@@ -175,6 +175,7 @@ type config struct {
 	Profile string `long:"profile" description:"Enable HTTP profiling on given port -- NOTE port must be between 1024 and 65535"`
 
 	DebugHTLC          bool `long:"debughtlc" description:"Activate the debug htlc mode. With the debug HTLC mode, all payments sent use a pre-determined R-Hash. Additionally, all HTLCs sent to a node with the debug HTLC R-Hash are immediately settled in the next available state transition."`
+	Debug	           bool `long:"debug" description:"Run lnd in debug mode."`
```

Comment:
Rather than use a config parameter for this, build flags in Go should be utilized instead. This path would involve defining a method in the rpc server area, then having two versions: one that actually deletes the channel, and one that's a no-op but gives an error.

Reply:
Personally, I favor option 1 as it doesn't require us copying over the entire config flag. In this case, the `ForgetChannel` RPC method would be the only one copied over in the short term. So the `ForgetChannel` RPC in `rpcserver.go`, would call out to a canned `forgetChannel` method declared at the top of the file. The two versions would then be properly gated on the build flag.

Reply:
Change made. But what I don't know is how to put that canned `forgetChannel` func at the top op `rpcserver.go`. I think I am forced to put it into a separate `rpcserver_production.go` file. Or how to do it otherwise?